### PR TITLE
added logger to test

### PR DESCRIPTION
**Pull request by DougVonMoser**
now logs a single test case ahead of running test suite. 
run it a couple of times to notice that people have to traverse 7 seats sometimes? 6 might have been the max only for algos 1 - 4

### CODE REVIEW

**Review by nrod80 — APPROVED** [review]
I'm so sorry for the late response! This looks so fucking cool, I love the directions on the output, it's totally awesome

**Comment by nrod80** [comment]
I will go ahead and allow you to do the honors of merging it 😜

**Comment by DougVonMoser** [comment]
better late than never

File: logger.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+module.exports = function(optimize){
+
+
+	// returns individual cost
+	function helperCost(name){
+		let origIdx = this.currentSeats.indexOf(name);
+		let finalIdx = this.finalPlacement.indexOf(name);
+		let result = Math.abs(finalIdx - origIdx)
+		if (result > 6) result = Math.abs( (result - 7) - 7 )
```
Comment (nrod80) [inline logger.js:20]: Please be more consistent with your use of semicolons in the future.

**Pull request merged**